src/audio/component.c

```diff
@@ -62,8 +62,8 @@ int comp_set_state(struct comp_dev *dev, int cmd)
 	int requested_state = comp_get_requested_state(cmd);
 
 	if (dev->state == requested_state) {
-		comp_info(dev, "comp_set_state(), state already set to %u",
-			  dev->state);
+		comp_dbg(dev, "comp_set_state(), state already set to %u",
+			 dev->state);
```

Comment:
not sure about this one. Indeed in some cases it isn't an error, but it has also proven useful sometimes when debugging. Maybe we should check all cases where this gets printed now and for all those cases check in the caller? Or add a wrapper `comp_set_state_if_different()` or something and then maybe even change this to `comp_warn()`

Reply:
ok, maybe warn is more suitable here, since it is not error, use info seems too light, afterall, it is not a right operation.